### ISSUE TITLE
Unknown plugin "transform-runtime" specified in "base" at 0

### ISSUE THREAD

**Issue by ramijarrar** [xolvio/chimp#282]
Running Mocha tests with the latest version of chimp (0.29.0) throws the following error on startup:

```sh
C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\dist\lib\mocha\mocha-wrapper.js:43
  throw e;
  ^

ReferenceError: Unknown plugin "transform-runtime" specified in "base" at 0, attempted to resolve relative to "C:\\Users\\nakula\\Desktop\\local-work\\chimprepro\\tests"
    at C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:206:17
    at Array.map (native)
    at Function.normalisePlugins (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:182:20)
    at OptionManager.mergeOptions (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:298:36)
    at OptionManager.init (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-core\lib\transformation\file\options\option-manager.js:486:10)
    at compile (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-register\lib\node.js:83:45)
    at loader (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-register\lib\node.js:128:14)
    at Object.require.extensions.(anonymous function) [as .js] (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\babel-register\lib\node.js:138:7)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\mocha\lib\mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\mocha\lib\mocha.js:213:14)
    at Mocha.run (C:\Users\nakula\AppData\Roaming\nvm\v5.4.0\node_modules\chimp\node_modules\mocha\lib\mocha.js:453:10)
Mocha failed
```

The previous version of chimp (0.28.2) runs without any issues.

**Comment by samhatoum** [comment]
try 0.29.1 :)

**Comment by ramijarrar** [comment]
Tried with 0.29.1 and getting the same error :(

**Comment by samhatoum** [comment]
would be awesome if you can create a small repro to help us find the issue

**Comment by ramijarrar** [comment]
I can confirm this issue is specific to Windows.

I think has something to do with it:
```sh
npm WARN optional Skipping failed optional dependency /chimp/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@0.3.8
```

**Comment by ramijarrar** [comment]
@samhatoum Can we reopen this?

**Comment by Sanjo** [comment]
Fixed with 0.30.0

**Comment by ramijarrar** [comment]
:+1:
